rules/windows/builtin/application/mssqlserver/win_mssql_failed_logon_from_external_network.yml

```diff
@@ -0,0 +1,49 @@
+title: MSSQL Server Failed Logon From External Network
+id: ebfe73c2-5bc9-4ed9-aaa8-8b54b2b4777d
+related:
+    - id: 218d2855-2bba-4f61-9c85-81d0ea63ac71
+      type: similar
+status: experimental
+description: Detects failed logon attempt from clients with external network IP to an MSSQL server. Can be a sign of a bruteforce attack.
```

Comment:
```suggestion
description: Detects failed logon attempts from clients with external network IP to an MSSQL server. This can be a sign of a bruteforce attack.
```